File: Server/src/main/java/com/api/server/foodb/Nutrient.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "nutrients")
public class Nutrient {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "legacy_id")
    private Integer legacyId;

    @Size(max = 255)
    @NotNull
    @Column(name = "type", nullable = false)
    private String type;

    @Size(max = 9)
    @NotNull
    @Column(name = "public_id", nullable = false, length = 9)
    private String publicId;

    @Size(max = 255)
    @NotNull
    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "export")
    private Boolean export;

    @Size(max = 255)
    @Column(name = "state")
    private String state;

    @Size(max = 255)
    @Column(name = "annotation_quality")
    private String annotationQuality;

    @Lob
    @Column(name = "description")
    private String description;

    @Size(max = 255)
    @Column(name = "wikipedia_id")
    private String wikipediaId;

    @Lob
    @Column(name = "comments")
    private String comments;

    @Size(max = 255)
    @Column(name = "dfc_id")
    private String dfcId;

    @Size(max = 255)
    @Column(name = "duke_id")
    private String dukeId;

    @Size(max = 255)
    @Column(name = "eafus_id")
    private String eafusId;

    @Lob
    @Column(name = "dfc_name")
    private String dfcName;

    @Size(max = 255)
    @Column(name = "compound_source")
    private String compoundSource;

    @Lob
    @Column(name = "metabolism")
    private String metabolism;

    @Lob
    @Column(name = "synthesis_citations")
    private String synthesisCitations;

    @Lob
    @Column(name = "general_citations")
    private String generalCitations;

    @Column(name = "creator_id")
    private Integer creatorId;

    @Column(name = "updater_id")
    private Integer updaterId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/Nutrient.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("nutrients")]
[Index("Name", Name = "index_nutrients_on_name", IsUnique = true)]
[Index("Name", "PublicId", Name = "index_nutrients_on_name_and_public_id", IsUnique = true)]
[Index("PublicId", Name = "index_nutrients_on_public_id", IsUnique = true)]
public partial class Nutrient
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("legacy_id")] public int? LegacyId { get; set; }

    [Column("type")] [StringLength(255)] public string Type { get; set; } = null!;

    [Column("public_id")]
    [StringLength(9)]
    public string PublicId { get; set; } = null!;

    [Column("name")] public string Name { get; set; } = null!;

    [Column("export")] public bool? Export { get; set; }

    [Column("state")] [StringLength(255)] public string? State { get; set; }

    [Column("annotation_quality")]
    [StringLength(255)]
    public string? AnnotationQuality { get; set; }

    [Column("description", TypeName = "mediumtext")]
    public string? Description { get; set; }

    [Column("wikipedia_id")]
    [StringLength(255)]
    public string? WikipediaId { get; set; }

    [Column("comments", TypeName = "mediumtext")]
    public string? Comments { get; set; }

    [Column("dfc_id")] [StringLength(255)] public string? DfcId { get; set; }

    [Column("duke_id")]
    [StringLength(255)]
    public string? DukeId { get; set; }

    [Column("eafus_id")]
    [StringLength(255)]
    public string? EafusId { get; set; }

    [Column("dfc_name", TypeName = "mediumtext")]
    public string? DfcName { get; set; }

    [Column("compound_source")]
    [StringLength(255)]
    public string? CompoundSource { get; set; }

    [Column("metabolism", TypeName = "mediumtext")]
    public string? Metabolism { get; set; }

    [Column("synthesis_citations", TypeName = "mediumtext")]
    public string? SynthesisCitations { get; set; }

    [Column("general_citations", TypeName = "mediumtext")]
    public string? GeneralCitations { get; set; }

    [Column("creator_id")] public int? CreatorId { get; set; }

    [Column("updater_id")] public int? UpdaterId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }
}